pyproject.toml

```diff
@@ -58,6 +58,9 @@ elasticsearch = "~8.12.0"
 numpy = "~1.26.0"
 mcp = "1.8.0"
 datamodel-code-generator = "0.30.0"
+stagehand-py = "~0.3.6"
+browserbase = "~1.4.0"
+rich = "~14.0.0"
```

Comment:
Are this libs not included in stagehand-py? If not, please add inline comments that they are mandatory for stagehand